airflow/operators/docker_operator.py

```diff
@@ -14,11 +14,12 @@
 
 import json
 
+from airflow.contrib.hooks.docker_hook import DockerHook
```

Comment:
Don't rely on contrib in core please. Move docker_hook  to core plz.

Reply:
Ok, will move it.

Reply:
Done.